apps/nextjs-website/src/lib/strapi/overviewsCodec.ts

```diff
@@ -102,7 +102,7 @@ const makeStrapiOverviewsPopulate = () =>
         populate: ['bottomLink', 'items.icon'],
       },
       tutorialSection: {
-        populate: ['tutorials.image'],
+        populate: ['tutorials.image', 'tutorials.product.logo'],
```

Comment:
I dont think `logo` is needed here.
```suggestion
        populate: ['tutorials.image', 'tutorials.product'],
```